deploy/helm/cht-chart/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
I think I'd prefer this whole `deploy` directory to be moved to `scripts` as a subdir.

Reply:
The idea was to have a directory only dealing with deployment and put different kinds of deployment options in here. Still want to move it?

Reply:
Yes I think so. Otherwise the top level ends up too wide. But I'm open to other suggestions!

Reply:
Addressed.